src/commands/index.js

```diff
@@ -98,6 +98,7 @@ module.exports = function(client) {
     client.hosting.clone = loadCommand("hosting-clone");
   }
   client.hosting.disable = loadCommand("hosting-disable");
+  client.hosting.channel = {};
```

Comment:
This would blow away all the other channel commands, wouldn't it? This might be the result of a bad merge (I know I cleaned this up again, making it alphabetized again)

Reply:
removed